src/GrumPHP/Task/PHPUnitCoverageTask.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace GrumPHP\Task;
+
+use GrumPHP\Configuration\GrumPHP;
+use GrumPHP\Runner\TaskResult;
+use GrumPHP\Task\Context\ContextInterface;
+use GrumPHP\Task\Context\GitPreCommitContext;
+use GrumPHP\Task\Context\RunContext;
+use GrumPHP\Task\TaskInterface;
+use InvalidArgumentException;
+use SimpleXMLElement;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class PHPUnitCoverageTask implements TaskInterface
```

Comment:
Maybe CloverCoverage is a better name for the task? The suffix Task is dropped in all the tasks, so no need for that one. Can you also add class docblocks?